adding a new troubleshooting issue for SNMP (#10060)
DataDog/documentation: content/en/network_monitoring/devices/troubleshooting.md

1. Try loosening ACLs/firewall rules for your devices.
2. Run `snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6` from the host your Agent is running on. If you get a timeout without any response, there is likely something blocking the Datadog Agent from collecting metrics from your device.

#### What do I do if Datadog supports a vendor or device type but my specific model isn’t supported? 

1. Contact [Datadog support][1] to put in a request to support your specific model.

> 2. Profiles can be extended to support additional `sysobjectid`’s 

Reviewer: ```suggestion
2. Profiles can be extended to support additional `sysobjectid` values. 
```

> 2. Profiles can be extended to support additional `sysobjectid`’s 

Reviewer: This shouldn't be an ordered list -- we don't expect people to do both these things, right? How about: "Contact support to request support for your model. Alternatively, extend the profile to supports your sysobjectid value. For example, to monitor another type of Cisco CSR, modify the ISR profile to list ..." Also. Where do they do this modification? In a file (which one)? On the command line?

    For example, if you want to monitor another type of Cisco CSR, you can modify the ISR profile directly to list another `sysobjectid` like this: 
    
    ```
		snmpwalk -v 2c -c [community string] [ip address] 1.3.6.1.2.1.1.2
    ```

> **Note**: If you do not know the `sysobjectid` of your device, first try googling for it, or run a `snmpwalk` on a host that can reach out to your device. Use the output to list the profile to match against. 

Reviewer 2: ```suggestion
**Note**: If you do not know the `sysobjectid` of your device, try doing an Internet search or run a `snmpwalk` on a host that can reach out to your device. Use the output to list the profile to match against. 
```
